docker-compose-dev.yml

```diff
@@ -0,0 +1,40 @@
+version: "3.7"
+services:
+
+  minio:
+    image: minio/minio # Stable minio
+    volumes:
+      - minio-data:/data
+    networks:
+      - xain-fl
+    environment:
+      MINIO_ACCESS_KEY: minio
+      MINIO_SECRET_KEY: minio123
+    command: server /data
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://localhost:9000/minio/health/live"]
+      interval: 30s
+      timeout: 20s
+      retries: 3
+    ports:
+      - "9000:9000"
+
+  xain-fl:
+    build:
+      context: .
+      dockerfile: Dockerfile.dev
+    command: coordinator -f test_array.npy
+    volumes:
+      - '.:/app'
```

Comment:
Do we need the whole repo to be mounted? The problem with that is that the container's `root` user might end up owning some files if it writes to them.

Reply:
We need two files that are at the root of the repo (README.md and test_array.npy) in order for pip to be able to install the packages and the coordinator to start, respectively. We can only volume-mount directories, thus, the root of the repo. 

We could leave this as-is, I could update the Dockerfile to use a custom username, where it makes sense, or we could investigate into tools like https://podman.io/

Reply:
> We can only volume-mount directories, thus, the root of the repo.

I think that we can actually mount individual files as well. This [SO answer](https://stackoverflow.com/a/42260979/1836144) suggests the only restriction is that you need to use an absolute path and that the file has to exist in the image (which can be done by `touch`ing it in the Dockerfile).

Anyway that's not a blocker.

Reply:
You are correct. 
Updated the volume mounts to only include necessary files.